tssc/step_implementers/shared/openscap_generic.py

```diff
@@ -0,0 +1,664 @@
+        #       need to be run in a privilaged mode
+        print()
+        print(f"Mount container: {container_name}")
+        container_mount_path = OpenSCAPGeneric.__buildah_mount_container(
+            buildah_unshare_comand=buildah_unshare_comand,
+            container_id=container_name
+        )
+        print(f"Mounted container ({container_name}) with mount path: '{container_mount_path}'")
+
+        # download the open scap input file
+        oscap_input_definitions_uri = self.get_config_value('oscap-input-definitions-uri')
+        print()
+        print(f"Download input definitions: {oscap_input_definitions_uri}")
+        oscap_input_file = download_and_decompress_source_to_destination(
+            source_url=oscap_input_definitions_uri,
+            destination_dir=self.get_working_dir()
+        )
+        print(f"Download input definitions to: {oscap_input_file}")
+
+        # if specified download oscap tailoring file
+        oscap_tailoring_file = None
+        oscap_tailoring_file_uri = self.get_config_value('oscap-tailoring-uri')
+        if oscap_tailoring_file_uri:
+            print()
+            print(f"Download oscap tailoring file: {oscap_tailoring_file_uri}")
+            oscap_tailoring_file = download_and_decompress_source_to_destination(
+                source_url=oscap_tailoring_file_uri,
+                destination_dir=self.get_working_dir()
+            )
+            print(f"Download oscap tailoring file to: {oscap_tailoring_file}")
+
+        # determine oscap eval type based on document type
+        print()
+        print(f"Determine OpenSCAP document type of input file: {oscap_input_file}")
+        oscap_document_type = OpenSCAPGeneric.__get_oscap_document_type(
+            oscap_input_file=oscap_input_file
+        )
+        print(
+            "Determined OpenSCAP document type of input file"
+            f" ({oscap_input_file}): {oscap_document_type}"
+        )
+        print()
+        print(
+            f"Determine OpenSCAP eval type for input file ({oscap_input_file}) "
+            f"of document type: {oscap_document_type}"
+        )
+        oscap_eval_type = OpenSCAPGeneric.__get_oscap_eval_type_based_on_document_type(
+            oscap_document_type=oscap_document_type
+        )
+        print(
+            f"Determined OpenSCAP eval type of input file ({oscap_input_file}): {oscap_eval_type}"
+        )
+
+        # Execute scan in the context of buildah unshare
+        #
+        # NOTE: run in the context of `buildah unshare` so that container does not
+        #       need to be run in a privilaged mode
+        oscap_out_file_path = self.write_working_file(f'oscap-{oscap_eval_type}-out')
+        oscap_xml_results_file_path = self.write_working_file(
+            f'oscap-{oscap_eval_type}-results.xml'
+        )
+        oscap_html_report_path = self.write_working_file(f'oscap-{oscap_eval_type}-report.html')
+        print()
```

Comment:
just put a \n in the next print, rather than taking the double hit to stdout

Reply:
fixed.